### ISSUE TITLE
Create a backup transient?

### ISSUE THREAD

**Issue by lukestateson** [scottsweb/wp-instagram-widget#119]
Currently when plugin successfully scrapes Instagram it creates the transient for HOUR_IN_SECONDS * 3 - 3 hours.

We need another "backup" transient every successful scrape set for at least 1 month (or even 1 year).

In case there will be an Instagram outage / server IP ban or any other issue plugin should load from the backup transient instead of showing the error.

**Comment by vchabal** [comment]
See previous closed issues #122 - The plugin is no longer maintained
As well issues #121 #120 #118

I replaced plugin with custom html, short example:
```
<div id="instasidebar"></div>
<script>
var insta_username = 'username_goes_here'
jQuery.getJSON({
    url: 'https://www.instagram.com/'+insta_username+'/?__a=1',
    success: function(data) {
        var posts = data.graphql.user.edge_owner_to_timeline_media.edges.slice(0, 9), 
            html = [];
        for (var i = 0; i < posts.length; i++) {
            html.push('<a href="https://www.instagram.com/p/' + posts[i].node.shortcode + '"><img src="' + posts[i].node.thumbnail_src + '" width=33.33% /></a>')
        }
        document.getElementById('instasidebar').innerHTML = html.join('');
    }
});
</script>
```

**Comment by scottsweb** [comment]
This project is being archived (background in #118). Instagram filed a trademark complaint which saw the plugin removed from WordPress.org and then proceeded to block it from accessing instagram.com. All issues are being closed and it will soon be in a read only state.
